src/Components/CreateImageWizard/steps/msAzure.beta.js

```diff
@@ -0,0 +1,255 @@
+      type: 'text',
+      label: 'Azure Tenant GUID',
+      isRequired: true,
+      autoFocus: true,
+      validate: [
+        {
+          type: validatorTypes.REQUIRED,
+        },
+        {
+          type: validatorTypes.PATTERN,
+          pattern:
+            /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i,
+          message: 'Please enter a valid tenant ID',
+        },
+      ],
+      condition: {
+        when: 'azure-type',
+        is: 'azure-type-manual',
+      },
+    },
+    {
+      component: componentTypes.TEXT_FIELD,
+      name: 'azure-tenant-id',
+      className: 'pf-u-w-50',
+      'data-testid': 'azure-tenant-id',
+      type: 'text',
+      label: 'Azure Tenant GUID',
+      isRequired: true,
+      isReadOnly: true,
+      condition: {
+        when: 'azure-type',
+        is: 'azure-type-source',
+      },
+    },
+    {
+      component: 'azure-auth-button',
+      name: 'azure-auth-button',
+      'data-testid': 'azure-auth-button',
+      required: true,
+      isRequired: true,
+    },
+    {
+      component: componentTypes.TEXT_FIELD,
+      name: 'azure-subscription-id',
+      className: 'pf-u-w-50',
+      'data-testid': 'azure-subscription-id',
+      type: 'text',
+      label: 'Subscription ID',
+      isRequired: true,
+      validate: [
+        {
+          type: validatorTypes.REQUIRED,
+        },
+        {
+          type: validatorTypes.PATTERN,
+          pattern:
+            /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i,
+          message: 'Please enter a valid subscription ID',
+        },
+      ],
+      condition: {
+        when: 'azure-type',
+        is: 'azure-type-manual',
+      },
+    },
+    {
+      component: componentTypes.TEXT_FIELD,
+      name: 'azure-subscription-id',
+      className: 'pf-u-w-50',
+      'data-testid': 'azure-subscription-id',
+      type: 'text',
+      label: 'Subscription ID',
+      isRequired: true,
+      isReadOnly: true,
+      condition: {
+        when: 'azure-type',
+        is: 'azure-type-source',
+      },
+    },
+    {
+      component: componentTypes.TEXT_FIELD,
+      name: 'azure-resource-group',
+      className: 'pf-u-w-50',
+      'data-testid': 'azure-resource-group',
+      type: 'text',
+      label: 'Resource group',
+      isRequired: true,
+      validate: [
+        {
+          type: validatorTypes.REQUIRED,
+        },
+        {
+          type: validatorTypes.PATTERN,
+          pattern: /^[-\w._()]+[-\w_()]$/,
+          message:
+            'Resource group names only allow alphanumeric characters, ' +
+            'periods, underscores, hyphens, and parenthesis and cannot end in a period',
+        },
+      ],
+      condition: {
+        when: 'azure-type',
+        is: 'azure-type-manual',
+      },
+    },
+    {
+      component: 'azure-resource-groups',
+      name: 'azure-resource-group',
+      className: 'pf-u-w-50',
```

Comment:
It doesn't seem like the `pf-u-w-50` class gets used for selects. I think it is picking up its width from .pf-c-select__toggle which is 100% by default.

Reply:
I've made it to work, if the className prop is passed to the select directly and not to the FormGroup, it works as expected (it's actually exactly what other components do as well anyways :)

I've then changed the class to `pf-u-max-width`, that you use in AWS, to be consistent.